eidas_node/proxy_service/views.py

```diff
@@ -114,7 +113,7 @@ def create_saml_request(self, issuer: str) -> SAMLRequest:
         :return: A SAML request.
         """
         # Replace the request id with the token id to find the request later during pairing with the response.
-        self.light_request.id = self.light_token.id
+        self.light_request.id = 'T' + self.light_token.id  # Prefix with 'T' because XML id cannot start with a digit.
```

Comment:
The prefix should probably be a constant of some sorts.

Reply:
`constants.TOKEN_ID_PREFIX`